### ISSUE TITLE
Use Machine Learning to Extract Information from Documents with Swagger UI

### ISSUE THREAD

**Issue by advit-ramesh** [SAPDocuments/Issues#192]
Tutorials: https://developers.sap.com/tutorials/cp-aibus-dox-swagger-ui.html
--------------------------

Write here what you need help with ...

When I try to invoke POST /document/jobs from swagger along with the sample invoice and JSON payload, I get the below error response

{
  "error": {
    "code": "E102",
    "message": "Request exceeds trial account quotas.",
    "details": []
  }
}

I created a new subaccount and I deployed the Document information extractor service from the booster. 

Can you please advise how I overcome this trial account quota?  Currently, this is impeding us from running a Proof of concept to a potential client.

Thanks 
Regards
Advit

**Comment by Juliana-Morais** [comment]
Hi Advit,

Please observe the Trial Account Input Limits (https://help.sap.com/viewer/5fa7265b9ff64d73bac7cec61ee55ae6/SHIP/en-US/44c9f41e36544ad0acf8796665668c96.html).

Best regards,
Juliana

**Comment by Juliana-Morais** [comment]
Hi Advit,

I've just heard we had an issue with our trial limits that has already been fixed.

We have done a root cause analysis. We found the issue which has been related to a fix that we have recently delivered. Now, it should work for a newly created trial account. Hence, I would like to ask you to create a new trial account and try it again. Thank you for providing feedback. We hope it will work again.

Best regards,
Juliana
